### DISCUSSION TITLE
compact模式下，支持单元格宽度最大值设置吗？

### DISCUSSION THREAD

**Discussion by ritawr** [antvis/S2#3035] [Q&A]
![image](https://github.com/user-attachments/assets/ce84b938-a762-4eb9-a889-35a68414e162)
我想列表在compact布局下，单元格宽度做个最大值限制，以免一些极端情况下，显示异常。看了文档和props没有发现对应的配置

**Comment by lijinke666** [comment]
文档没写就是不支持, 欢迎 PR
